it/controllers/employment/AddEmploymentControllerISpec.scala

```diff
@@ -279,15 +278,15 @@ class AddEmploymentControllerISpec extends IntegrationTest with ViewHelpers with
             dropEmploymentDB()
             authoriseAgentOrIndividual(user.isAgent)
             userDataStub(IncomeTaxUserData(None), nino, taxYear)
-            urlPost(url(validTaxYear2021), follow = false, body = yesNoFormYes, welsh = user.isWelsh, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(validTaxYear2021)))
+            urlPost(url(taxYearEOY), follow = false, body = yesNoFormYes, welsh = user.isWelsh, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
           }
 
           "status SEE_OTHER" in {
             result.status shouldBe SEE_OTHER
           }
 
           "redirect to employer name page" in {
-            result.header(HeaderNames.LOCATION).toString.contains("/update-and-submit-income-tax-return/employment-income/2021/employer-name?employmentId=") shouldBe true
+            result.header(HeaderNames.LOCATION).getOrElse("") contains employerNameUrlWithoutEmploymentId(taxYearEOY).getOrElse("") shouldBe true
```

Comment:
can probably leave this as location contains employerNameUrl.. etc

Reply:
as if both getOrElses happen (not that it would) the test would still pass as "" == ""

Reply:
I kinda wanna make it so it looks for the employerNameUrlWithoutEmploymentId, because this was in a few places so if that url got updated it would need changing in a few places. A .get would be safe her right because both the LOCATION and employerNameUrlWithoutEmploymentId are always containing a value?

Reply:
I would have thought so yeah